datasafe-encryption/datasafe-encryption-impl/src/main/java/de/adorsys/datasafe/encrypiton/impl/cmsencryption/CMSEncryptionServiceImpl.java

```diff
@@ -6,8 +6,8 @@
 import de.adorsys.datasafe.encrypiton.impl.cmsencryption.decryptors.Decryptor;
 import de.adorsys.datasafe.encrypiton.impl.cmsencryption.decryptors.DecryptorFactory;
 import de.adorsys.datasafe.encrypiton.impl.cmsencryption.exceptions.DecryptionException;
-import de.adorsys.datasafe.encrypiton.impl.keystore.generator.ProviderUtils;
 import de.adorsys.datasafe.types.api.context.annotations.RuntimeDelegate;
+import de.adorsys.keymanagement.bouncycastle.adapter.services.deprecated.generator.ProviderUtils;
```

Comment:
You should not use classes from `deprecated` package. ProviderUtils should be defined in datasafe

Reply:
ok